app/src/server/tasks/generateTestSetEntry.task.ts

```diff
@@ -20,6 +20,8 @@ import {
 import { getOpenaiCompletion } from "../utils/openai";
 import defineTask from "./defineTask";
 import { queueHeadToHeadEvalJobsForTestingEntry } from "./evaluateTestSetEntries.task";
+import { RateLimitError } from "openai";
```

Comment:
Nit: import order

Reply:
We can fix this by adding an eslint rule to automatically sort imports after you get your changes merged.